{"name": "test_login[case18]", "status": "passed", "attachments": [{"name": "log", "source": "7ea9de09-87b9-4571-9cd2-1a36d4ca5b35-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case1", "value": "{'case_id': 9, 'title': '连续输入密码错误5次3', 'data': '(\"J/fY3PE2HH0kvMZzbv5BLQ==\",\"9i3jjLm3SxkpHakKWnTasg==\",\"6666\",\"$2y$10$reArgS92v9zKSj.Z77yls.nqw0mxi1gVyEVzu.eYv9vT6/2DS8Z9q\",\"username\")', 'expected': '{\"code\": 0, \"msg\": \"账户密码错误\"}'}"}], "start": 1714393854602, "stop": 1714393857681, "uuid": "a8d2b6ef-9de6-48a3-a306-1ab091588b1b", "historyId": "2a3cc3aec06c2ef1ed11635d255fe689", "testCaseId": "ee3b72303bf8fc3659dea4eab1f6b87e", "fullName": "test_ddt2.TestDdt#test_login", "labels": [{"name": "suite", "value": "test_ddt2"}, {"name": "subSuite", "value": "TestDdt"}, {"name": "host", "value": "LAPTOP-Q5VMTL8G"}, {"name": "thread", "value": "25284-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_ddt2"}]}